src/status_im/contexts/communities/actions/invite_contacts/view.cljs

```diff
@@ -0,0 +1,130 @@
+(ns status-im.contexts.communities.actions.invite-contacts.view
+  (:require
+    [quo.core :as quo]
+    [quo.foundations.resources :as resources]
+    [quo.theme]
+    [react-native.core :as rn]
+    [react-native.gesture :as gesture]
+    [react-native.share :as share]
+    [status-im.common.contact-list-item.view :as contact-list-item]
+    [status-im.common.contact-list.view :as contact-list]
+    [status-im.contexts.communities.actions.invite-contacts.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- no-contacts-view
+  [{:keys [theme id]}]
+  (let [customization-color             (rf/sub [:profile/customization-color])
```

Comment:
Unnecessary re-renders we should avoid 👍🏼, but here it's a bit different @ibrkhalil.

The customization color is not the cause of re-renders by itself. But if the user changes the profile color in another device while this component is rendered, the mobile UI should reflect that.

Most of the time subs should be inside the renderer, except if you intentionally want to make the component non-reactive, which is a valid solution, but only if performance is a problem or you know the value is stable (e.g. an ID), otherwise the app becomes buggy because we break React's & re-frame's model of `ui = f(states)`